src/pages/malynar/verifikacia/[[...params]].tsx

```diff
@@ -0,0 +1,13 @@
+import {NextPage} from 'next'
+
+import {PageLayout} from '../../../components/PageLayout/PageLayout'
+
+const Verifikacia: NextPage = () => {
+  return (
+    <PageLayout title="Verifikácia" contentWidth={2}>
+      Verifikačný e-mail bol odoslaný na zadanú e-mailovú adresu.
```

Comment:
V buducnosti by som ocakaval, ze tam bude nejake spolocna komponenta ulozena v `src/components` ktora bude importnuta to vsetkych seminarov.... mozno by som uz komponent obsahujuci len jeden div urobil a budeme to mat pripravene na stylovanie v buducnosti